http-servers/src/main/java/com/palantir/remoting/http/server/JerseyComponents.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting.http.server;
+
+public final class JerseyComponents {
```

Comment:
there is also the brave business

On Wed, Jul 13, 2016 at 3:42 PM Mark Elliot <notifications@github.com>
wrote:

> In
> http-servers/src/main/java/com/palantir/remoting/http/server/JerseyComponents.java
> <https://github.com/palantir/http-remoting/pull/119#discussion_r70694358>:
>
> > + * Licensed under the Apache License, Version 2.0 (the "License");
> > + * you may not use this file except in compliance with the License.
> > + * You may obtain a copy of the License at
> > + *
> > + *     http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +package com.palantir.remoting.http.server;
> > +
> > +public final class JerseyComponents {
>
> Isn't that what this is?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/119/files/02a27ca486636ecf6d369cb7ca9cf26136c2fff2#r70694358>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGOdwepSsBCqJq0H6atax7s2h6tCohQwks5qVT-HgaJpZM4Ig2Io>
> .
>